src/main/java/cubicchunks/world/column/Column.java

```diff
@@ -710,7 +714,8 @@ public boolean isTerrainPopulated() {
 		//with cubic chunks the whole column is never fully generated,
 		//So some heuristic is needed to tell vanilla is generator is populated here
 		//for now - tell it that it is if any cube is populated
-		return this.cubeMap.all().stream().anyMatch(c -> c.getGeneratorStage().isLastStage());
+		
+		return this.cubeMap.all().stream().anyMatch(c -> (c.getGeneratorStage() == GeneratorStage.LIVE));
```

Comment:
The GeneratorPipeline is not available in this content. Columns exist on the client, therefore the world visible to the column does not necessarily have the pipeline. Having the stage know if it is the final stage would be a better solution.